src/components/Publish/_validation.ts

```diff
@@ -54,13 +55,23 @@ const validationService = {
   })
 }
 
+const decimalDigitsValidation = new RegExp(/^\d+(\.\d{1,6})?$/)
 const validationPricing = {
   type: Yup.string()
     .matches(/fixed|dynamic|free/g, { excludeEmptyString: true })
     .required('Required'),
   // https://github.com/jquense/yup#mixedwhenkeys-string--arraystring-builder-object--value-schema-schema-schema
   price: Yup.number()
     .min(1, (param: { min: number }) => `Must be more or equal to ${param.min}`)
+    .max(
+      1000000,
+      (param: { max: number }) => `Must be less than or equal to ${param.max}`
+    )
+    .test(
+      'maxDigitsAfterDecimal',
+      'Must have less than 7 decimal digits',
```

Comment:
there's `MAX_DECIMALS` in `src/@utils/constants.ts` where this number should come from